build.psm1

```diff
@@ -446,6 +446,31 @@ Fix steps:
         Pop-Location
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected: Feature PSCommandNotFoundSuggestion to be Enabled
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\ExperimentalFeature.Basic.Tests.ps1: line 47
47:             $expFeature.Enabled | Should -BeEnabled -Name $expFeature.Name
```